core/tests/webdriverio_utils/forms.js

```diff
@@ -665,10 +663,12 @@ var CodeMirrorChecker = function(elem, codeMirrorPaneToScroll) {
     var actualDiffDict = {};
     var scrollBarWebElement = null;
     if (codeMirrorPaneToScroll === 'first') {
-      scrollBarWebElement = await scrollBarElements[0];
+      var scrollBarElements = await $$('.CodeMirror-vscrollbar');
+      scrollBarWebElement = scrollBarElements[0];
     } else {
+      var scrollBarElements = await $$('.CodeMirror-vscrollbar');
```

Comment:
There was error in previous code (I have not used await with `$$`), that why I am modifying this. Also if we will use this at top then $$ is returning undefined because in starting (when this function is called) the element is not present, so in order to get the elment we need to use it before the place where it is used.

Reply:
Can we not declare it above the if condition then in order to avoid duplicacy?

Reply:
Done.